packages/server/src/app/models/auth/auth.router.js

```diff
@@ -1,51 +1,54 @@
 const express = require('express')
 const limiter = require('express-rate-limit')
-const { matchedData } = require('express-validator')
+const { matchedData, checkSchema } = require('express-validator')
 const AuthController = require('./auth.controller')
-const { authValidation } = require('./auth.validation')
-const UserController = require('../user/user.controller')
-const { cookieJwtAuth } = require('../../middleware/cookie-jwt-auth')
-const JwtService = require('../../services/jwt.service')
+const confirmPasswordValidator = require('../../middleware/confirm-password.validator')
+const jwtAuthHeaderValidator = require('../../middleware/jwt-auth-header-validator')
+const { loginSchema, newUserSchema } = require('../../validation/user.validator')
 
 const router = express.Router()
 
-// On the login and register routes, allow maximum 3 requests per 5 minutes
+// On the login and register routes, allow maximum 10 requests per 5 minutes
 const authLimiter = limiter({
   windowMs: 5 * 60 * 1000,
-  max: 3,
+  max: 10,
   standardHeaders: true,
   legacyHeaders: false
 })
 
-router.get('/', cookieJwtAuth, async (request, response, next) => {
-  try {
-    response.send({ user: await UserController.instance().getById(request.user.id) })
-  } catch (error) {
-    next(error)
+router.post(
+  '/register',
+  authLimiter,
+  checkSchema(newUserSchema, ['body']),
+  confirmPasswordValidator,
+  async (request, response, next) => {
+    try {
+      const createdUser = await AuthController.instance().register(matchedData(request))
+
+      response.status(201).json(createdUser)
+    } catch (error) {
+      next(error)
+    }
   }
-})
-
-router.post('/', authLimiter, authValidation, async (request, response, next) => {
-  try {
-    const user = await AuthController.instance().auth(matchedData(request))
-    const token = JwtService.instance().generateToken(user.toJSON(), '8h')
-
-    const expiryDate = new Date(Date.now() + 60 * 60 * 8000) // 8 hour
-    response.cookie('auth-token', token, { expires: expiryDate })
-
-    response.status(200).send({ user, message: `Welkom terug ${user.username}! 😎` })
-  } catch (error) {
-    next(error)
+)
+
+router.post(
+  '/login',
+  authLimiter,
+  checkSchema(loginSchema, ['body']),
+  async (request, response, next) => {
+    try {
+      const token = await AuthController.instance().login(matchedData(request))
+
+      response.header('Authorization', `Bearer ${token}`).send()
```

Comment:
You might want to add a message that tells you're logged in

Reply:
We can tell by if the log in action was successful by looking at the status of the response